### PR TITLE
Update browserlist supported browsers matrix

### PR DESCRIPTION
**Pull request by ntwb**
WordPress supported browsers: https://make.wordpress.org/design/handbook/design-guide/browser-support/

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -7,10 +7,13 @@
 					"last 2 Chrome versions",
 					"last 2 Firefox versions",
 					"last 2 Safari versions",
+					"last 2 Edge versions",
+					"last 2 Opera versions"
```
Comment (swissspidy) [inline .babelrc]: You're missing a comma here

**Review by swissspidy — CHANGES_REQUESTED** [review]

File: .babelrc (hunk 1 above)
Comment (ntwb) [inline .babelrc]: Hehehe, copy and paste fail 😝

**Comment by aduth** [comment]
>WordPress supported browsers: https://make.wordpress.org/design/handbook/design-guide/browser-support/

The referenced article doesn't mention any of the browsers added here? Edge makes sense to me, of course. Similarly, Opera (assuming it continues to receive updates) and `1%` seem as "good ideas". Is that the intent of these changes? Should we seek to update the browser support handbook document to reflect Edge and Opera too, then?

**Comment by ntwb** [comment]
Yes, the handbook should be updated, the specifics of the browsers not mentioned are detailed further in this ticket: [#WP37651](https://core.trac.wordpress.org/ticket/37651)

**Review by aduth — APPROVED** [review]

**Pull request merged**